e2e/npm_packages/packages/lib/BUILD.bazel

```diff
@@ -0,0 +1,59 @@
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@jest//:defs.bzl", "jest_test")
+
+npm_link_all_packages(name = "node_modules")
+
+# Basic js with 1 npm dep
+js_library(
+    name = "lib_jslib",
+    srcs = ["index.js"],
+    deps = [
+        "//:node_modules/@aspect-test/a",
+    ],
+)
+
+# Basic npm package wrapping the js
+npm_package(
+    name = "lib",
+    srcs = [
+        ":lib_jslib",
+        "package.json",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+# Library of tests
+js_library(
+    name = "lib_tests",
+    srcs = glob(["*.spec.js"]),
+    deps = [
+        ":lib_jslib",
+        "//:node_modules/uvu",
+    ],
+)
+
+# Jest test runners
+jest_test(
+    name = "test_cjs",
+    config = "//:jest.config.cjs",
+    data = [
+        ":lib_tests",
+        "babel.config.json",
+        "//:jest.config.cjs-deps",  # must manually declare jest_test(config) deps
```

Comment:
IMO this is a bug or confusing API. The `jest_test(config)` will often have deps such as jest plugins/config/setup etc.

WDYT?

Reply:
suppose we could have the config attribute that a `js_library` instead that specifies the deps? could be a follow-up addition

Reply:
oh. config here is already a js_library. Why not just pull the deps out of that js_library in this PR then?

Reply:
See the last commit